devices/wholeBodyDynamics/WholeBodyDynamicsDevice.cpp

```diff
@@ -975,6 +973,14 @@ bool WholeBodyDynamicsDevice::loadSettingsFromConfig(os::Searchable& config)
     if (settings.useJointVelocity) { yInfo() << "wholeBodyDynamics: jointVelFilterCutoffInHz: " <<   settings.jointVelFilterCutoffInHz << " Hz."; }
     if (settings.useJointAcceleration) { yInfo() << "wholeBodyDynamics: jointAccFilterCutoffInHz: " <<   settings.jointAccFilterCutoffInHz << " Hz."; }
 
+    // Set time to check for a new temperature measurement. The default value is 0.55;
+    // is set in the device constructor
+    if( prop.check("checkTemperatureEvery_seconds") &&
+        prop.find("checkTemperatureEvery_seconds").isDouble())
+    {
+        checkTemperatureEvery_seconds = prop.find("checkTemperatureEvery_seconds").asDouble();
+    }
```

Comment:
This is unrelated to the bug, but this configuration was being handled in the wrong place as well.

Reply:
Updated changelog to specify this fix as well.